essentials/src/utils.rs

```diff
@@ -15,17 +15,30 @@
 // along with polkadot-introspector.  If not, see <http://www.gnu.org/licenses/>.
 //
 
-use std::time::Duration;
-
+use clap::Parser;
 use log::info;
+use std::time::Duration;
 use thiserror::Error;
 use tokio::time::sleep;
 
-use crate::constants::{RETRY_COUNT, RETRY_DELAY_MS};
+#[derive(Clone, Debug, Parser)]
+#[clap(
+	rename_all = "kebab-case",
+	allow_external_subcommands = true, // HACK: to parse it as a standalone config
+)]
+pub struct RetryOptions {
+	/// Max of times to retry a failed connection before giving up.
+	#[clap(name = "retry", default_value = "3", long)]
```

Comment:
Isn't this too low ? RPC nodes sometimes can be overloaded.

Reply:
Changed to 10